doc/classes/@GDScript.xml

```diff
@@ -609,34 +669,53 @@
 			</return>
 			<description>
 				Print a stack track at code location, only works when running with debugger turned on.
+				Output in the console would look something like this:
+				[codeblock]
+				Frame 0 - res://test.gd:16 in function '_process'
+				[/codeblock]
 			</description>
 		</method>
 		<method name="printerr" qualifiers="vararg">
 			<return type="void">
 			</return>
 			<description>
 				Print one or more arguments to strings in the best way possible to standard error line.
+				[codeblock]
+				printerr("prints to stderr")
+				[/codeblock]
 			</description>
 		</method>
 		<method name="printraw" qualifiers="vararg">
 			<return type="void">
 			</return>
 			<description>
 				Print one or more arguments to strings in the best way possible to console. No newline is added at the end.
+				[codeblock]
+				printraw("A","B")
+				[/codeblock]
+				Prints AB
```

Comment:
IMO, this doesn't show the difference between `print` and `printraw`.

Reply:
Your right I think I meant to have two print statements with one output line.